src/Translations.js

```diff
@@ -3,12 +3,19 @@
 
 const ENGLISH_TRANSLATION = {
     "Papyros": "Papyros",
-    "enter_code": "Enter your code here",
-    "enter_input": "Provide input for your code here",
-    "code_output": "Code output",
+    "code": "Code",
+    "code_placeholder": "Enter your code here",
```

Comment:
```suggestion
    "code_placeholder": "Write your code here and click run to execute...",
```

If possible, make this "JavaScript code" or "Python code" depending on the mode.